docs/earth-command/earth-command.md

```diff
@@ -170,8 +170,6 @@ For example:
 * `--git-url-instead-of='git@github.com:=https://github.com/'` forces use of SSH-based URLs for GitHub (default)
 * `--git-url-instead-of='https://github.com/=git@github.com:'` forces use of HTTPS-based URLs for GitHub
 
-Currently, only one such pattern is allowed to be specified.
-
```

Comment:
Perhaps we should remove these options from here. Maybe we just mention that the setting has moved to the config file and not give out any other details here.

Reply:
I agree, that'll simplify things.

Reply:
I think it's still worth mentioning that the options have moved. Example:

```markdown
##### `--git-url-instead-of <git-instead-of>`	

Also available as an env var setting: `GIT_URL_INSTEAD_OF=<git-instead-of>`.

This option is now deprecated. Please use <insert-page-name-and-link-here> instead.
```